internal/pkg/sylog/sylog.go

```diff
@@ -159,6 +170,11 @@ func SetLevel(l int) {
 	loggerLevel = messageLevel(l)
 }
 
+// DisableColor for the logger
+func DisableColor() {
```

Comment:
It's probably easier in this function to just do this:

```
messageColors = map[messageLevel]string{
	fatal: "",
	error: "",
	warn: "",
	info: "",
}

colorReset = ""
```

This way we don't need all of the extra conditionals scattered in the code.

Reply:
I like this better too.